src/communities/communities.controllers.js

```diff
@@ -1,7 +1,42 @@
 import { status } from '../../config/response.status.js';
 import { response } from '../../config/response.js';
+import { createCommunityService } from './communities.service.js';
 import { getCommunitiesService } from './communities.service.js';
 
+
+export const createCommunityController = async (req, res, next) => {
+    const { userId, bookId, address, tag, capacity, createdAT, updatedAt  } = req.body;
+
+    // 누락된 파라미터 확인
+    const missingParams = [];
+    if (!userId) missingParams.push('userId');
+    if (!bookId) missingParams.push('bookId');
+    if (!address) missingParams.push('address');
+    if (!capacity) missingParams.push('capacity');
+
+    // 누락된 정보가 있을 경우
+    if (missingParams.length > 0) {
+        return res.status(400).send({
+            isSuccess: false,
+            code: "COMMON001",
+            message: `필요한 정보가 누락되었습니다: ${missingParams.join(', ')}`
+        });
+    }
+
+    try {
+        // 커뮤니티 생성 및 DTO 반환
+        const communityData = await createCommunityService(userId, bookId, address, tag, capacity, createdAT, updatedAt );
+
+        // 성공 응답 전송
+        res.status(status.CREATED.status).send(response(
+            status.CREATED,
+            { community: communityData }
+        ));
```

Comment:
이 부분 반환되는 데이터가 굳이 필요하지 않을 거 같고 (모임 생성되면 모임 다시 조회하는 화면으로 가서 api가 재 호출됩니다.) 그냥 바로 res.send()로 저희가 정의해둔 상태 코드 반환해주시면 돼요!

```suggestion
        res.send(response(status.CREATED));
```

Reply:
수정했습니다! 감사해요